### PR TITLE
[cleanup] Re-generated sample app migrations #119

### LINKED ISSUE #119: [cleanup] Re-generate sample app migrations
We don't need to keep migrations for the sample app since it's meant only for testing, therefore we can delete all the migrations and regenerate them, except for `0002_default_permissions.py` or any other data migration, which shall left intact.

### PR DESCRIPTION
**Pull request by codesankalp**
Regenerated all migrations except data migration.

Closes #119

### CODE REVIEW

**Comment by codesankalp** [comment]
@nemesisdesign The tests and QA-checks are passing locally. I have attached some images regarding this:

image

image

image

**Comment by nemesisdesign** [comment]
@atb00ker @pandafy @devkapilbansal did we have this problem of the build failing because of EULA acceptance somewhere else right, did we? If yes, how did we solve it?

**Comment by devkapilbansal** [comment]
> @atb00ker @pandafy @devkapilbansal did we have this problem of the build failing because of EULA acceptance somewhere else right, did we? If yes, how did we solve it?

Yes this problem arises when mysql was updated.

@codesankalp consider removing `sudo apt-get upgrade` from here
https://github.com/openwisp/openwisp-firmware-upgrader/blob/6c01c1ec2b224789c1c1a1fb37ab1adf4f2189af/.github/workflows/ci.yml#L39

File: tests/openwisp2/sample_firmware_upgrader/migrations/0001_initial.py [hunk 1]
```diff
@@ -133,12 +155,41 @@ class Migration(migrations.Migration):
...
+                            ),
+                            (
+                                'ar71xx-generic-xd3200-squashfs-sysupgrade.bin',
+                                'YunCore XD3200',
+                            ),
+                            (
+                                'ar71xx-generic-ubnt-airrouter-squashfs-sysupgrade.bin',
+                                'Ubiquiti AirRouter',
+                            ),
+                        ],
```
Comment (nemesisdesign) [inline tests/openwisp2/sample_firmware_upgrader/migrations/0001_initial.py:192]: this line should not be changed

File: tests/openwisp2/sample_firmware_upgrader/migrations/0001_initial.py [hunk 2]
```diff
@@ -148,11 +199,16 @@ class Migration(migrations.Migration):
                     'build',
                     models.ForeignKey(
                         on_delete=django.db.models.deletion.CASCADE,
-                        to=swapper.get_model_name('firmware_upgrader', 'Build'),
+                        to='sample_firmware_upgrader.build',
```
Comment (nemesisdesign) [inline tests/openwisp2/sample_firmware_upgrader/migrations/0001_initial.py:202]: same here

File: tests/openwisp2/sample_firmware_upgrader/migrations/0001_initial.py [hunk 3]
```diff
@@ -203,23 +259,22 @@ class Migration(migrations.Migration):
                         blank=True,
                         null=True,
                         on_delete=django.db.models.deletion.CASCADE,
-                        to=swapper.get_model_name(
-                            'firmware_upgrader', 'BatchUpgradeOperation'
-                        ),
+                        to='sample_firmware_upgrader.batchupgradeoperation',
```
Comment (nemesisdesign) [inline tests/openwisp2/sample_firmware_upgrader/migrations/0001_initial.py:262]: same here

File: tests/openwisp2/sample_firmware_upgrader/migrations/0001_initial.py [hunk 4]
```diff
@@ -203,23 +259,22 @@ class Migration(migrations.Migration):
...
                     ),
                 ),
                 (
                     'device',
                     models.ForeignKey(
                         on_delete=django.db.models.deletion.CASCADE,
-                        to=swapper.get_model_name('config', 'Device'),
```
Comment (nemesisdesign) [inline tests/openwisp2/sample_firmware_upgrader/migrations/0001_initial.py:215]: same here, applies to all other occurrences

**Review by nemesisdesign — CHANGES_REQUESTED** [review]
Avoid changing lines which call specific functions from swapper or calls to dynamic data structures, see below.

File: tests/openwisp2/sample_firmware_upgrader/migrations/0001_initial.py [hunk 5]
```diff
@@ -15,8 +19,8 @@ class Migration(migrations.Migration):
     initial = True
 
     dependencies = [
-        ('config', '0025_update_device_key'),
-        ('openwisp_users', '0007_unique_email'),
+        migrations.swappable_dependency(settings.CONFIG_DEVICE_MODEL),
+        ('openwisp_users', '0011_user_first_name_150_max_length'),
```
Comment (devkapilbansal) [inline tests/openwisp2/sample_firmware_upgrader/migrations/0001_initial.py:23]: @codesankalp can we change this dependency with AUTH_USER_MODEL?
```suggestion
        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
```

**Review by devkapilbansal — CHANGES_REQUESTED** [review]
@codesankalp Looks good, just one minor change

**Review by nemesisdesign — APPROVED** [review]

**Comment by nemesisdesign** [comment]
Thanks @codesankalp @devkapilbansal :+1:

**Pull request merged**